### ISSUE TITLE
[Ulduar] Ancient Gate of the Keepers bug

### ISSUE THREAD

**Issue by UldLover** [dalaranwow/dalaran-wow#5597]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I was doing attune run for Ulduar 10 (ID 1173) and the door before General Vezax bugged out. All keepers are dead, but Hodir's door seems to be iced over still. The instance actually first bugged at Mimiron with encounter in progress bug, so we just stop for the night. Sadly door bug is not fixed in time.

![wowscrnshot_090518_111221](https://user-images.githubusercontent.com/43009177/45107531-7fad1a00-b0ff-11e8-8c22-ca7638b162ec.jpg)
![wowscrnshot_090518_111346](https://user-images.githubusercontent.com/43009177/45107532-8045b080-b0ff-11e8-8fe1-879fb13393de.jpg)
![wowscrnshot_090518_111447](https://user-images.githubusercontent.com/43009177/45107533-8045b080-b0ff-11e8-8cbb-268b62ec20cf.jpg)
![wowscrnshot_090518_111839](https://user-images.githubusercontent.com/43009177/45107534-8045b080-b0ff-11e8-838f-1ec1e2a83ee0.jpg)
![wowscrnshot_090518_112150](https://user-images.githubusercontent.com/43009177/45107535-8045b080-b0ff-11e8-8de7-7d8e1473bb48.jpg)
![wowscrnshot_090518_112151](https://user-images.githubusercontent.com/43009177/45107536-8045b080-b0ff-11e8-95ac-4d81b7876475.jpg)

**Expected behaviour**: Tell us what should happen instead
The door before General Vezax should open regardless of keepers kill order. 

**Steps to reproduce the problem**:
1.  Wipe on Mimiron then try to enter instance. 
2.  Wait for soft reset then kill all keepers
3.  Try to go thru Ancient gate of the keepers

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by LoveDalaranWow** [comment]
Happens pretty much every week. People are abusing summoning stones to work around this bug.

**Comment by Lothloryen** [comment]
I am on it, I deployed some corrections and thought it was fixed already. It seems it is not.

**Comment by Lothloryen** [comment]
Hello there,

This issue should not happen anymore, if anyone experiences it, please let me know.

**Comment by djhype** [comment]
This happened to us tonight. All 4 keepers dead, but door wouldn't unlock. Had to get a Warlock to glitch through and summon us.

Raid Instance ID: 613

**Comment by djhype** [comment]
@Lothloryen This happened to us tonight. All 4 keepers dead, but door wouldn't unlock. Had to get a Warlock to glitch through and summon us.

Raid Instance ID: 613

**Comment by Lothloryen** [comment]
Hello there,

Since I have been unable to reproduce this issue by myself locally, I have been deploying corrections to the live server to see if they fixed it. With today's, seems the issue is finally fixed (fingers crossed). If anyone experiences the glitch, please let me know.

Thank you.
